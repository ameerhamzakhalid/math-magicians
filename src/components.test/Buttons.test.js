import React from 'react';
import { render } from '@testing-library/react';
import Buttons from '../components/Buttons';

describe('Buttons', () => {
  it('should render the button with the correct label', () => {
    const btnLabel = 'Click Me';
    const { getByText } = render(<Buttons btn={btnLabel} id="my-button" clickEvent={() => {}} />);
    const button = getByText(btnLabel);
    expect(button).toBeTruthy();
  });
});
