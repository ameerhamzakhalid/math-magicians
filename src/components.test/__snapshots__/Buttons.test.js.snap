// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Buttons matches the snapshot 1`] = `
<DocumentFragment>
  <button
    class="button"
    id="my-button"
    type="button"
  >
    Click Me
  </button>
</DocumentFragment>
`;
